index.md

```diff
@@ -64,7 +64,113 @@ Some of the below sessions will occur on our Discord server and some will occur
+
+18. [Controllable and Interpretable Singing Voice Decomposition via Assem-VC](https://neuripscreativityworkshop.github.io/2021/accepted/srw_58.pdf)
+    * *Kang-Wook Kim, Junhyeok Lee*
+
+19. [BERTian Poetics: Constrained Composition with Masked LMs](https://neuripscreativityworkshop.github.io/2021/accepted/srw_59.pdf)
+    * *Christopher Akiki, Martin Potthast*
+
+20. [Towards Lightweight Controllable Audio Synthesis with Conditional Implicit Neural Representations](https://neuripscreativityworkshop.github.io/2021/accepted/srw_70.pdf)
+    * *Jan Zuiderveld, Marco Federici, Erik J Bekkers*
+
+21. [Inspiration Retrieval for Visual Exploration](https://neuripscreativityworkshop.github.io/2021/accepted/srw_76.pdf)
+    * *Nihal Jain, Praneetha Vaddamanu, Paridhi Maheshwari, Vishwa Vinay, Kuldeep Kulkarni*
+
+22. [Extending the Vocabulary of Fictional Languages using Neural Networks](https://neuripscreativityworkshop.github.io/2021/accepted/srw_78.pdf)
+    * *Thomas Zacharias, Ashutosh Taklikar, Raja Giryes*
+
+23. [Steerable discovery of neural audio effects](https://neuripscreativityworkshop.github.io/2021/accepted/srw_82.pdf)
+    * *Christian J. Steinmetz, Joshua D. Reiss*
+
+24. [XCI-Sketch: Extraction of Color Information from Images for Generation of Colored Outlines and Sketches](https://neuripscreativityworkshop.github.io/2021/accepted/srw_85.pdf)
+    * *V Manushree, Sameer Saxena, Parna Chowdhury, Manisimha Varma Manthena, Harsh Rathod, Ankita Ghosh, Sahil S Khose*
+
+25. [Exploring Latent Dimensions of Crowd-sourced Creativity](https://neuripscreativityworkshop.github.io/2021/accepted/srw_88.pdf)
+    * *Umut Kocasari, Alperen Bag, Efehan Atici, Pinar Yanardag*
+
+26. [Ethics and Creativity in Computer Vision](https://neuripscreativityworkshop.github.io/2021/accepted/srw_89.pdf)
+    * *Negar Rostamzadeh, Emily Denton, Linda Petrini*
+
+27. [Evolving Evocative 2D Views of Generated 3D Objects](https://neuripscreativityworkshop.github.io/2021/accepted/srw_93.pdf)
+    * *Eric Chu*
+
+28. [RiverGAN: Fluvial Landscapes Generation with Conditional GAN and Physical Simulations](https://neuripscreativityworkshop.github.io/2021/accepted/srw_94.pdf)
+    * *Xun Liu, Runjia Tian*
+
+29. [Sculpting with Words](https://neuripscreativityworkshop.github.io/2021/accepted/srw_98.pdf)
+    * *Joel Simon, Victor Perez Muñoz, Tal Shiri*
+
+30. [Neural Design Space Exploration](https://neuripscreativityworkshop.github.io/2021/accepted/srw_99.pdf)
+    * *Wei Jiang, Richard Davis, Kevin Gonyop Kim, Pierre Dillenbourg*
+
+31. [Inspiration through Observation: Demonstrating the Influence of Generated Text on Creative Writing](https://neuripscreativityworkshop.github.io/2021/accepted/srw_102.pdf)
+    * *Melissa Roemmele*
+
+32. [Gaudí: Conversational Interactions with Deep Representations to Generate Image Collections](https://neuripscreativityworkshop.github.io/2021/accepted/srw_104.pdf)
```

Comment:
`srw` should be replaced with `ncw` no?

Reply:
Oops, mixed up the acronyms for the two workshops I am organizing.